application/views/admin/survey/Question/question_view.php

```diff
@@ -200,8 +200,12 @@
                             </td>
                             <td>
                                 <?php
-                                    if (isset($aAdvancedSetting['expression']) && $aAdvancedSetting['expression']==2){
-                                        LimeExpressionManager::ProcessString('{' . $aAdvancedSetting['value'] . '}', $qid);
+                                    if (isset($aAdvancedSetting['expression']) && $aAdvancedSetting['expression'] > 0) {
```

Comment:
`if ($aAdvancedSetting['expression']) `?

1. It must be always set : if not set : someone broke system ;) then it's cool if we have broken CI
2. More clear